frontend/scripts/react-components/dashboard-element/dashboard-indicators/dashboard-indicators.container.jsx

```diff
@@ -32,17 +30,12 @@ const mapStateToProps = state => ({
 class DashboardIndicatorsContainer extends React.PureComponent {
   static propTypes = {
     indicators: PropTypes.array,
-    getData: PropTypes.func.isRequired,
     activeIndicatorsList: PropTypes.array,
     dynamicSentenceParts: PropTypes.array,
     setActiveId: PropTypes.func.isRequired,
     removeActiveId: PropTypes.func.isRequired
   };
 
-  componentDidMount() {
-    this.props.getData('attributes');
```

Comment:
This was needed after all, the indicators list is not being loaded now. Let's discuss about the best way to fix this.